pkg/apis/certificates/v1beta1/conversion.go

```diff
@@ -24,15 +24,21 @@ import (
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
 	// Add field conversion funcs.
-	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("CertificateSigningRequest"),
-		func(label, value string) (string, string, error) {
-			switch label {
-			case "metadata.name",
-				"spec.signerName":
-				return label, value, nil
-			default:
-				return "", "", fmt.Errorf("field label not supported: %s", label)
-			}
-		},
-	)
+	for _, kind := range []string{"CertificateSigningRequest", "ClusterTrustBundle"} {
+		err := scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind(kind),
```

Comment:
Nit: I think this sharing is a little too clever --- CertificateSigningRequest and ClusterTrustBundle are separate types, and the fact that they currently have identical field->label conversion funcs is just accidental.

Can we just write out both definitions separately?  Otherwise someone who evolves this in the future may be tempted to start adding `if`s inside the loop.

Reply:
I originally had it written exactly that way but then I thought that future us must be better than to just shove an additional `if` in there, and so I switched to the loop logic :smile: 

I'll move it back, I think it makes sense to treat different objects differently :slightly_smiling_face: